pkgs/development/libraries/rocksdb/default.nix

```diff
@@ -27,9 +27,18 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-vRPyrXkXVVhP56n5FVYef8zbIsnnanQSpElmQLZ7mh8=";
   };
 
-  nativeBuildInputs = [ cmake ninja ];
```

Comment:
This commit kind of looks broken to what it claims to be. Did you forget to add something to the git index? 
I have the feeling its a bit messed up with what was pushed here.

https://github.com/NixOS/nixpkgs/pull/305974